src/Dev-12/src/test/java/FirefoxTestLoginMailRu.java

```diff
@@ -0,0 +1,116 @@
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ *
+ */
+public class FirefoxTestLoginMailRu {
+    private static final String WEBDRIVER_CHROME_DRIVER = "webdriver.chrome.driver";
+    private static final String CHROMEDRIVER_EXE_PATH = ".\\chromedriver\\chromedriver.exe";
```

Comment:
DRY, to many copy paste and re-usage of the same properties in other classes for other browsers. Create base class for tests and implement these filelds there with public getters.